.github/workflows/turbopack-update-tests-manifest.yml

```diff
@@ -62,7 +62,10 @@ jobs:
           node-version: ${{ env.NODE_LTS_VERSION }}
           check-latest: true
 
-      - run: corepack enable
+      - name: Setup corepack
+        run: |
+          npm i -g corepack@0.31
```

Comment:
isn't corepack included with pnpm? is this a newer version to fix a bug?

Reply:
I believe the issue here was a change in npm package signatures that broke older versions of corepack: https://github.com/nodejs/corepack/issues/612

Pnpm includes corepack-like functionality (but not corepack itself) in recent versions if you don't already have corepack enabled, as long as you bootstrap it by installing any recent version of pnpm globally first. Instead of bootstrapping pnpm, we instead choose to bootstrap corepack here. IMO, either is fine.